### ISSUE TITLE
Can multiple clock settings be simulated against a single trace?

### ISSUE THREAD

**Issue by drajarshi** [accel-sim/accel-sim-framework#116]
Regarding the trace-driven mode, the .trace file is generated when I run my application at a certain clock frequency (e.g. f1) on the GPU. Subsequently the .traceg file is also generated.

Now, if I specify a clock frequency f2 (different from f1) in the gpgpusim.config and run the simulation using the  .traceg file generated from the run at frequency f1, is it a valid simulation run? 
Or am I allowed to only set f1 as the frequency in the gpgpusim.config for the purpose of running the simulation? 
In other words, if I want to run a simulation at a clock frequency of f2, do I need to have a .trace file generated on the GPU at the same frequency f2?
I am thinking the purpose of the simulation is to allow different frequency settings in the gpgpusim.config and run using a single trace, but wanted to get a confirmation on it.

Thanks.

**Comment by JRPan** [comment]
Hi, 

Simulation config file clock rate does not have to be the same as the clock rate that was used to collect traces. 

So you are correct. 

Thanks

**Comment by drajarshi** [comment]
Thanks @JRPan 
Closing issue.
